libs/web/shared/data-access/models/project.json

```diff
@@ -10,5 +10,5 @@
       }
     }
   },
-  "tags": []
+  "tags": ["type:ui","scope:web"]
```

Comment:
question: `type:ui` doesn't seems to match the `data-access/models` stuff. 

@nartc Your input?

Reply:
Yes that is correct, that is a mistake on my behalf